### PR TITLE
[MIG] iot_input_oca: Migration to 13.0

### PR DESCRIPTION
**Pull request by olgamarcocb**
Standard Migration

Change of module name

Depends on #27

### CODE REVIEW

File: iot_input_oca/views/iot_device_input_views.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    </record>
+    <record id="iot_device_input_form" model="ir.ui.view">
+        <field name="name">iot.device.input.form</field>
+        <field name="model">iot.device.input</field>
+        <field name="arch" type="xml">
+            <form>
+                <header />
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
```
Comment (etobella) [inline iot_input_oca/views/iot_device_input_views.xml]: The new logic is made using the web_ribbon

**Review by etobella — APPROVED** [review]
Just  a minor change on a view. Review on runbot made.

Thanks for the changes on the test, it is easier to understand.

Approved but waiting for the change

File: iot_input_oca/views/iot_device_input_views.xml (hunk 1 above)
Comment (olgamarcocb) [reply iot_input_oca/views/iot_device_input_views.xml]: Thank you! :+1:

**Pull request merged**